netebpfext/ebpf_ext_attach_provider.h

```diff
@@ -46,6 +47,7 @@ _Releases_lock_(registration) void ebpf_ext_attach_leave_rundown(
 ebpf_result_t
 ebpf_ext_attach_register_provider(
     _In_ const ebpf_attach_type_t* attach_type,
+    _In_ const ebpf_program_type_t* program_type,
```

Comment:
ebpf_program_load puts program_type before attach_type.  We should use consistent order of arguments.